src/wp-includes/comment.php

```diff
@@ -3345,7 +3357,11 @@ function _prime_comment_caches( $comment_ids, $update_meta_cache = true ) {
 	if ( ! empty( $non_cached_ids ) ) {
 		$fresh_comments = $wpdb->get_results( sprintf( "SELECT $wpdb->comments.* FROM $wpdb->comments WHERE comment_ID IN (%s)", implode( ',', array_map( 'intval', $non_cached_ids ) ) ) );
 
-		update_comment_cache( $fresh_comments, $update_meta_cache );
+		update_comment_cache( $fresh_comments, false );
+	}
+
+	if ( $update_meta_cache ) {
```

Comment:
Similar to `_prime_term_caches` function we have to add `@since 6.3.0 Use wp_lazyload_comment_meta() for lazy-loading of term meta.` document.

Reply:
Fixed.